go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/google/uuid v1.3.0
 	github.com/hashicorp/go-cty v1.4.1-0.20200414143053-d3edf31b6320
 	github.com/hashicorp/go-version v1.4.0
-	github.com/hashicorp/hcp-sdk-go v0.18.0
+	github.com/hashicorp/hcp-sdk-go v0.18.1-0.20220506085537-4a8c08ea0676
```

Comment:
just pumped to 0.19.0, so you can pin to that: https://github.com/hashicorp/hcp-sdk-go/releases/tag/v0.19.0